src/domain/services/logger.service.ts

```
@@ -0,0 +1,27 @@
+  warn: (
+    description: string,
+    params?: LoggerService.Params,
+  ) => Promise<LoggerService.Model>;
+  error: (
+    description: string,
+    params?: LoggerService.Params,
+  ) => Promise<LoggerService.Model>;
+
+  send: () => Promise<null>;
+
+  setLabels: (labels: Object) => void;
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)